src/screens/SSICredentialsRequiredScreen/index.tsx

```diff
@@ -23,6 +24,8 @@ import {
 import {ScreenRoutesEnum, StackParamList} from '../../types';
 import {getMatchingUniqueVerifiableCredential, getOriginalVerifiableCredential} from '../../utils/CredentialUtils';
 import {toCredentialSummary} from '../../utils/mappers/credential/CredentialMapper';
+import { SubmissionRequirementMatch } from "@sphereon/pex/dist/main/lib/evaluation/core/submissionRequirementMatch";
```

Comment:
What's up with this import?

Reply:
It's coming from selectResult.matches

Reply:
can you not use?

import { SubmissionRequirementMatch } from "@sphereon/pex";

Reply:
Yeah, the question obviously wasn't about why there is an import, it was about why the package wasn't imported

Reply:
Fixed, no idea where IJ got that the first time, just did ctrl-space